### ISSUE TITLE
Add Leader Pivot to Blazorise

### ISSUE THREAD

**Issue by sam-wheat** [Megabit/Blazorise#3412]
I wrote a [pivot table for Blazor](https://github.com/leaderanalytics/LeaderPivot.Blazor).  I wrote the control primarily to support another app I am writing.  I am not a UX developer.  I could never make the control look as good as Blazorize controls.  However you could probably make it look great in no time at all.  You are welcome to add it to Blazorize.

Regards,
Sam

**Comment by stsrki** [comment]
Hi @sam-wheat thank you for allowing us to use your code. Leader Pivot seems like a good component and we might incorporate some parts of it into our own #3214 version which is still WIP.

One thing I'm concerned about is the licensing. LGPL is not compatible with Apache 2.0. Is there any chance you could lose the LGPL?

**Comment by stsrki** [comment]
Yes, it can be used as a DLL. But I was understanding you wish for us to incorporate some parts of code into Blazorise? Am I wrong?

**Comment by sam-wheat** [comment]
You can use it in any way that is convenient to you.  My purpose in choosing LGPL is to prevent a third party from hijacking the code and making something of it that is not free for the rest of the community.  If you also use it in a commercial product that is fine with me.  I request that you mention/link to me and Leader Analytics in the code and documentation.

I suspect that you will want to reference the [base control](https://github.com/leaderanalytics/LeaderPivot) and create a Blazor wrapper around it as I have done with [LeaderPivot.Blazor](https://github.com/leaderanalytics/LeaderPivot.Blazor).  If you need fixes/enhancements we can work together to implement them via PR.  Used in this fashion you are truly only referencing a .dll and you can leverage the components that have already been written for Blazorise.  With that said - however you choose to use it up to you.

**Comment by David-Moreira** [comment]
Hello @sam-wheat 
I'm not sure a wrapper will give us enough flexibility to freely keep developing and expanding upon your pivot grid. 

And having to go through an additional PR over at **LeaderPivot** for new features / fixes is an additional step in the development process I believe we do not want to take. Specially when we do have a commercial side of the product, where we want to maintain an agile support of the product.

As @stsrki said, it would be more convenient for us if we could incorporporate some of the parts of your code into **Blazorise**, with, of course, the mentions to the original creator of the code, you.
